models/bamf_nnunet_mr_liver/config/default.yml

```diff
@@ -0,0 +1,42 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet MR Liver segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunner
+- BamfProcessorRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: 'mr'
```

Comment:
I suggest using the `%Modality` placeholder here, our DicomImporter module then will infer the Modality from the DICOM headers automatically, preventing e.g. CT scans from being processed.

Suggested change:  `mod: '%Modality'`.

**Note, that quotes are required as the string starts with a special character `%`.**